randovania/games/samus_returns/logic_database/Area 4 - Crystal Mines.txt

```diff
@@ -615,6 +666,11 @@ Extra - asset_id: collision_camera_009
   > Tunnel to Gamma Arena
       Morph Ball and Varia Suit
 
+> TODO: future dock; Heals? False
+  * Layers: default
+  > Door to Green Cystal Dugout
```

Comment:
I just noticed this room is spelled wrong lol. Should be `Green Crystal Dugout` 😛

Reply:
out of scope for this PR :P